### DISCUSSION TITLE
Rendering of localized classification store is slow in backend for many languages

### DISCUSSION THREAD

**Discussion by hbauer** [pimcore/pimcore#16392] [General]
In a current project, we need a localizable Classification Store that is displayed in a separate tab in a product-class. In the tab, for example, are 19 groups with 140 attributes from the Classification Store displayed - for 14 languages.
The problem is the poor performance when rendering the attributes because as soon as the tab is clicked, it takes approx. 6-8 seconds until the tab is activated and the content appears. For other products with more attributes it can take 20-30 seconds. Switching between languages takes only 0.5 seconds.

It seems that not only the content of the current language but for all languages is rendered. The more languages are added, the longer it takes. I tried to set "deferredRenderer: true" in the panel config but without any effect.

Tested with the current Pimcore v11.1.4, but the problem also occurs in Pimcore 10.6.

Does anyone else have the same problem or a solution?

**Comment by fashxp** [comment]
Hmm, maybe it is possible to tweak the extjs rendering a bit and defer the rendering until the tab is opened?

**Reply by robertSt7** [reply]
@hbauer @fashxp  please have a look at  https://github.com/pimcore/admin-ui-classic-bundle/pull/395
